manifests/static.pp

```diff
@@ -11,6 +11,6 @@
   pulpcore::admin { 'collectstatic --noinput':
     refreshonly => true,
     subscribe   => Concat['pulpcore settings'],
-    require     => File[$pulpcore::pulp_static_root],
+    require     => File[$pulpcore::static_root],
```

Comment:
```suggestion
    subscribe     => File[$pulpcore::static_root],
```
As discussed on IRC, there is a very small possibility that owner/group could be changed. In that case we would want to re-run collectstatic so that the collected static content will also be changed to match.

In most regular use cases that is unlikely to happen, but that also means the cost of making this change (in terms of execution time) is negligible since the refresh would only get triggered in that edge case.